apim-operator/pkg/apis/wso2/v1alpha1/zz_generated.openapi.go

```diff
@@ -217,6 +220,106 @@ func schema_pkg_apis_wso2_v1alpha1_RateLimitingStatus(ref common.ReferenceCallba
 	}
 }
 
+func schema_pkg_apis_wso2_v1alpha1_Security(ref common.ReferenceCallback) common.OpenAPIDefinition {
+	return common.OpenAPIDefinition{
+		Schema: spec.Schema{
+			SchemaProps: spec.SchemaProps{
+				Description: "Security is the Schema for the securities API",
+				Properties: map[string]spec.Schema{
+					"kind": {
+						SchemaProps: spec.SchemaProps{
+							Description: "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds",
```

Comment:
looks these descriptions are too long? Is it possible to wrap them without braking syntax?

Reply:
yes this is auto-generated file